src/drivers/ports/esp32/can_port.i

```diff
@@ -39,6 +39,59 @@
 static struct fs_counter_t rx_channel_overflow;
 static struct fs_counter_t errors;
 
+extern xSemaphoreHandle thrd_idle_sem;
+
+/**
+ * Check if module is waiting for bus idle.
+ *
+ * Note 3 to Status register description in SJA1000 reference:
+ *     "If both the receive status and the transmit status bits are
+ *      logic 0 (idle) the CAN-bus is idle. If both bits are set the
+ *      controller is waiting to become idle again..."
+ */
+static int is_waiting_for_bus_idle(struct can_driver_t *self_p)
```

Comment:
is 'status' here a value of Status register? 

By the way: all fields in esp32_can_t struct (mcus\esp32\esp32.h) are defined as uint32_t. But all registers in SJA1000 are 8 bit length. Is it OK?